conda/envs/noble_like/pixi.toml

```diff
@@ -6,20 +6,22 @@ authors = ["Jose Luis Rivero <jrivero@openrobotics.org>"]
 channels = ["conda-forge"]
 platforms = ["linux-64", "linux-aarch64", "win-64"]
 
+[target.win-64.activation.env]
+QT_QPA_PLATFORM_PLUGIN_PATH="%cd%\\.pixi\\envs\\default\\Library\\lib\\qt6\\plugins\\platforms"
+QML2_IMPORT_PATH="%cd%\\.pixi\\envs\\default\\Library\\lib\\qt6\\qml"
```

Comment:
nit: Can we use `CONDA_PREFIX` here?

https://pixi.sh/dev/reference/environment_variables/#environment-variables-set-by-pixi